pkg/common/providers/ocmprovider/cluster.go

```diff
@@ -217,6 +221,31 @@ func (o *OCMProvider) LaunchCluster(clusterName string) (string, error) {
 	return resp.Body().ID(), nil
 }
 
+func (o *OCMProvider) findRecycledCluster(cluster *v1.Cluster) string {
+	query := fmt.Sprintf("properties.InstalledVersion='%s' and properties.UpgradeVersion='' and properties.Status='%s'", cluster.Version().ID(), "completed-passing")
+	log.Println(query)
```

Comment:
Is this for debugging? Or do we always want this printed?

Reply:
It was debugging, will remove. Thanks~~~